.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 
 node_js:
+  - iojs
+  - '0.12'
```

Comment:
from the travis CI documentation:

> we encourage users to specify node and/or iojs to run using the latest stable releases.

so i would keep this to "node"

also we have seen a significant perf degradation with 0.12 and that will make tests take even longer.